ztf_viewer/pages/lc_csv.py

```diff
@@ -18,10 +19,24 @@ def get_csv(dr, oids, min_mjd=None, max_mjd=None):
         oid_df = pd.DataFrame.from_records(lc)
         oid_df["oid"] = oid
         oid_df["filter"] = meta["filter"]
+
+        try:
+            ref = ztf_ref.get(oid, dr)
+        except NotFound:
```

Comment:
Why could it happen? I think we more likely have the case when IRSA is unavailable, and I would prefer to still have "ref" / "ref_err" columns with "none" or "NaN" or some other special values